core/tests/protractor_utils/GetStartedPage.js

```diff
@@ -37,7 +37,8 @@ var GetStartedPage = function() {
       throw new Error('Unsupported tag type specified: ' + type);
     }
 
-    return await tag.getAttribute('content');
+    await waitFor.visibilityOf(tag, 'Tag is taking too long to appear');
+    return tag.getAttribute('content');
```

Comment:
Does this need an `await`?

Reply:
Actually, I am not sure, I tried following the points in the issue description, where we use `await` with `waitFor`. I'll try removing that.

Reply:
I think kevintab95 meant that we need an `await` for `tag.getAttribute('content');`. You definitely do need to `await` the `waitFor`